### PR TITLE
plotjuggler: fix layout param

### PR DESCRIPTION
**Pull request by gregjhogan**

### CODE REVIEW

File: tools/plotjuggler/juggle.py [hunk 1]
```diff
@@ -66,12 +66,12 @@ def start_juggler(fn=None, dbc=None, layout=None):
...
   if fn is not None:
-    extra_args += f"-d {fn}"
+    extra_args += f" -d {fn}"
   if layout is not None:
-    extra_args += f"-l {layout}"
+    extra_args += f" -l {layout}"
 
-  subprocess.call(f'plotjuggler --plugin_folders {INSTALL_DIR} {extra_args}',
-                  shell=True, env=env, cwd=juggle_dir)
+  cmd = f'plotjuggler --plugin_folders {INSTALL_DIR}{extra_args}'
```
Comment (adeebshihadeh) [inline tools/plotjuggler/juggle.py:73]: why remove the space between INSTALL_DIR and extra_args?

**Comment by adeebshihadeh** [comment]
Let's add a test case for the layouts in test_plotjugger.py

File: tools/plotjuggler/juggle.py (hunk 1 above)
Comment (gregjhogan) [reply tools/plotjuggler/juggle.py:73]: because then we have a trailing space with no extra params and two spaces before the first param

**Comment by gregjhogan** [comment]
feel free to take this one over if any additional changes are desired before merging

**Comment by adeebshihadeh** [comment]
Hmm, I guess the way the tests are written now wouldn't catch the bug. I'll merge for now.

**Pull request merged**